server/api/index.js

```diff
@@ -0,0 +1,61 @@
+const express = require("express");
```

Comment:
This file should be called something like grapql so now deploys it at api/graphql. It shouldn't be index js because we could have lots of different functions here

Reply:
Done